test/yoloe/test_yoloe_ai.py

```
@@ -0,0 +1,28 @@
+from pathlib import Path
+
+import pandas as pd
+
+from camvidlog2.vid import generate_frames_cv2
+from camvidlog2.yoloe.ai import generate_tracked_bboxes
+
+
+def test_generate_tracked_bboxes_with_real_video(video_path: Path):
```

Ruff (0.11.9)

Missing return type annotation for public function `test_generate_tracked_bboxes_with_real_video`

Add return type annotation: `None`

(ANN201)